test/quantization/fx/test_equalize_fx.py

```diff
@@ -11,17 +12,91 @@
     _convert_equalization_ref
 )
 
-from torch.testing._internal.common_quantization import NodeSpec as ns
-from torch.testing._internal.common_quantization import QuantizationTestCase
+from torch.testing._internal.common_quantization import (
+    NodeSpec as ns,
+    QuantizationTestCase,
+    SingleLayerLinearModel,
+    TwoLayerLinearModel,
+)
 
 # Standard Libraries
+import copy
 import numpy as np
 
 # Testing utils
 from hypothesis import given
 from hypothesis import strategies as st
 
 
+# Test with two connected linear layers
+class Linear2FP32Module(nn.Module):
+    def __init__(self):
+        super().__init__()
+        self.linear1 = nn.Linear(5, 5)
+        self.linear2 = nn.Linear(5, 5)
+
+    def forward(self, x):
+        x = self.linear1(x)
+        x = torch.add(x, 5)
```

Comment:
Is it intended to have an add here since the model name says Linear2FP32Module

Reply:
Sorry, I think the comment above is incorrect. I meant to say that it has 2 linear layers with an add operation in between. Would the name `LinearAddModule` be better?